src/uct/self/self_md.c

```diff
@@ -45,7 +54,7 @@ static ucs_status_t uct_self_md_open(const char *md_name, const uct_md_config_t
 
 UCT_MD_COMPONENT_DEFINE(uct_self_md, UCT_SELF_NAME,
                         uct_self_query_md_resources, uct_self_md_open, NULL,
-                        ucs_empty_function_return_unsupported,
-                        ucs_empty_function_return_unsupported, "SELF_",
+                        ucs_empty_function_return_success,
```

Comment:
rkey_unpack should return some dummy value in *handle_p argument, not just be empty